include/bluetooth/bluetooth.h

```diff
@@ -881,6 +886,167 @@ int bt_le_per_adv_start(struct bt_le_ext_adv *adv);
  */
 int bt_le_per_adv_stop(struct bt_le_ext_adv *adv);
 
+struct bt_le_per_adv_sync_synced_info {
+	/** Advertiser LE address and type. */
+	const bt_addr_le_t *addr;
+
```

Comment:
You need to add the SID here, otherwise how will the user match trains from the same device but with different SIDs? I know there's a different object for each sync scanner, but you either put both address and SID or none, in which case you need a `get_info()` function to retrieve them.

Reply:
Good catch! Added SID to the callback structs.